src/libraries/System.Diagnostics.DiagnosticSource/tests/System.Diagnostics.DiagnosticSource.Tests.csproj

```diff
@@ -39,19 +40,24 @@
     <Compile Include="PropagatorTests.cs" />
     <Compile Include="TagListTests.cs" />
   </ItemGroup>
+
+  <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp'">
+    <Compile Include="DiagnosticSourceEventSourceBridgeTests.cs" />
+    <Compile Include="RuntimeMetricsTests.cs" />
+    <Compile Include="TestNotSupported.cs" />
+  </ItemGroup>
+
   <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework'">
     <Compile Include="HttpHandlerDiagnosticListenerTests.cs" />
     <Compile Include="ActivityDateTimeTests.cs" />
     <Compile Include="$(CommonTestPath)System\Net\Configuration.cs" Link="Common\System\Net\Configuration.cs" />
     <Compile Include="$(CommonTestPath)System\Net\Configuration.Http.cs" Link="Common\System\Net\Configuration.Http.cs" />
   </ItemGroup>
-  <ItemGroup Condition="$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net9.0'))">
-    <Compile Include="RuntimeMetricsTests.cs" />
-  </ItemGroup>
+
   <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework'">
-    <Reference Include="System.Net.Http" />
-  </ItemGroup>
-  <ItemGroup>
     <ProjectReference Include="..\src\System.Diagnostics.DiagnosticSource.csproj" />
+
+    <Reference Include="System.Net.Http" />
```

Comment:
Why remove the condition on this item?  It looks like that might be the only thing that changed in this project?  Lots of unnecessary churn here.

Reply:
Oh - nevermind - I see you conditioned the projectreference